### PR TITLE
Upgrade to dataflows 1.1.0 and fix all tests broken by the change.

### PR DESCRIPTION
**Pull request by jakeakopp**

### CODE REVIEW

File: src/main/java/com/google/cloud/genomics/dataflow/readers/bam/BAMShard.java [hunk 1]
```diff
@@ -106,7 +106,6 @@ public long approximateSizeInBytes() {
 
   @Override
   public String toString() {
-    return file + ": " + contig.toString() + ", locus size = " + sizeInLoci() + 
-        ", span size = " + approximateSizeInBytes();
```
Comment (jakeakopp) [inline src/main/java/com/google/cloud/genomics/dataflow/readers/bam/BAMShard.java]: approximateSizeInBytes updates a cached value (as does span.approximateSizeInBytes), so calling toString was causing the object to be mutated. This seemed like a reasonable solution to me.

File: src/main/java/com/google/cloud/genomics/dataflow/readers/bam/Reader.java [hunk 2]
```diff
@@ -180,9 +180,11 @@ void processRecord(SAMRecord record) {
   
   void dumpStats() {
     timer.stop();
+    long elapsed = timer.elapsed(TimeUnit.MILLISECONDS);
```
Comment (jakeakopp) [inline src/main/java/com/google/cloud/genomics/dataflow/readers/bam/Reader.java]: This is not strictly related, but at one point I managed to break things in such a way that I got 0ms as the total time. This crashed, so I decided to fix it.

File: src/test/java/com/google/cloud/genomics/dataflow/pipelines/VariantSimilarityITCase.java [hunk 3]
```diff
@@ -53,8 +54,7 @@
  * See also http://maven.apache.org/surefire/maven-failsafe-plugin/examples/single-test.html
  */
 public class VariantSimilarityITCase {
-  
-  static final String OUTPUT_SUFFIX = "-00000-of-00001";
```
Comment (jakeakopp) [inline src/test/java/com/google/cloud/genomics/dataflow/pipelines/VariantSimilarityITCase.java]: The test started outputting 17 shards for the cloud case (although only one actually has content), so I adjusted the tearDown and read bits, and removed the setUp, since we no longer know beforehand exactly what files it will write.

File: src/main/java/com/google/cloud/genomics/dataflow/readers/bam/BAMShard.java (hunk 1 above)
Comment (iliat) [inline src/main/java/com/google/cloud/genomics/dataflow/readers/bam/BAMShard.java]: Can you change it to use cached value instead and if cached value not set then don't output it ?
That way if the computation was used in some decision making then cachedvalue is not zero and will be outputted and if not then nobody cares about it.

**Comment by iliat** [comment]
LGTM (modulo my toString comment) , I'll let @deflaux give her seal of approval.

File: src/main/java/com/google/cloud/genomics/dataflow/readers/bam/BAMShard.java (hunk 1 above)
Comment (jakeakopp) [inline src/main/java/com/google/cloud/genomics/dataflow/readers/bam/BAMShard.java]: Done.

**Comment by deflaux** [comment]
Great updates - LGTM - thanks so much @jakeakopp !
